content/docs/jsx-in-depth.md

```diff
@@ -404,9 +404,9 @@ function ListOfTenThings() {
 
 Children passed to a custom component can be anything, as long as that component transforms them into something React can understand before rendering. This usage is not common, but it works if you want to stretch what JSX is capable of.
 
-### Booleans, Null, and Undefined Are Ignored {#booleans-null-and-undefined-are-ignored}
+### Booleans, Null, Undefined and Empty Lists Are Ignored {#booleans-null-and-empty-lists-undefined-are-ignored}
```

Comment:
It's not really ignoring empty lists. `[undefined]` is also not rendering. I'm not sure we need to mention lists here since React isn't rendering the list but rendering each item of the list.

Reply:
Good point, so I'll close this PR :) Thanks for the advising!